addons/planner_project/static/src/js/planner_project.js

```diff
@@ -0,0 +1,63 @@
+(function(){
+    "use strict";
+    var _t = openerp.web._t;
+
+    openerp.planner.PlannerDialog.include({
+        prepare_planner_event: function() {
+            var self = this;
+            this._super.apply(this, arguments);
+            if(self.planner['planner_application'] == 'planner_project') {
+                var stages = {
+                    'development_process': [
```

Comment:
for each method (development_process, marketing_department, ....) should it be better to have a dict matching the id of the input with the value ? 
Such as `{'input_element_stage_1' : _t('validation'), 'input_element_stage_2' : _t('Development'), .... }`.
I think this change can also be apply to planner_crm ;)